<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_signUp</class>
 <widget class="QDialog" name="Dialog_signUp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>469</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-120</x>
     <y>250</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="username_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>40</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string> Username</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string> Name</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="email_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string> Email</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password_input">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>90</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string> Passowrd</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog_signUp</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog_signUp</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
